pkg/util/flag/tristate.go

```diff
@@ -56,10 +56,10 @@ func (f *BoolFlag) Set(value string) error {
 	return nil
 }
 
-func (f BoolFlag) Provided() bool {
+func (f Tristate) Provided() bool {
 	return f.provided
 }
 
-func (f *BoolFlag) Type() string {
+func (f *Tristate) Type() string {
 	return "bool"
```

Comment:
Probably shouldn't return "bool" here? Maybe "tri"? But I'm not sure what consumes this.